From c1ef98276065687ec13d2cf1df7d691d7448eefd Mon Sep 17 00:00:00 2001
From: s <s@Penelope.local>
Date: Mon, 24 Jan 2022 08:15:44 +0200
Subject: [PATCH] option 'curr' for :plot/:plotedit commands introduced

---
 src/actions/plot.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/actions/plot.c b/src/actions/plot.c
index e308435..5713ed8 100644
--- a/src/actions/plot.c
+++ b/src/actions/plot.c
@@ -65,7 +65,7 @@ int plotedit(wchar_t * s) {
     char command[BUFFERSIZE];
 
     if (! wcscmp(s, L"line") || ! wcscmp(s, L"scatter") ||
-        ! wcscmp(s, L"pie")  || ! wcscmp(s, L"bar")) {
+        ! wcscmp(s, L"pie")  || ! wcscmp(s, L"bar") || ! wcscmp(s, L"curr")) {
         char buffer[PATHLEN + 5];
         char path_out[PATHLEN];
         char type[BUFFERSIZE];
@@ -124,7 +124,7 @@ int plot(char * s, int r, int c, int rf, int cf) {
     char plug_buffer[PATHLEN];
     sprintf(command, "gnuplot -e \"filename='%s'\"", datafile);
     if (! strcmp(s, "line") || ! strcmp(s, "scatter") ||
-        ! strcmp(s, "pie")  || ! strcmp(s, "bar")) {
+        ! strcmp(s, "pie")  || ! strcmp(s, "bar") || ! strcmp(s, "curr")) {
         sprintf(buffer, "plot_%s", s);
         if (! plugin_exists(buffer, strlen(buffer), plug_buffer)) {
             sc_error("could not load default plotline file");
-- 
2.7.4 (Apple Git-66)

